.travis.yml

```diff
@@ -1,11 +1,10 @@
-sudo: required
```

Comment:
Hmm, it looks like this didn't help the startup time any. Maybe it needs to explicitly be `sudo: false`?